docs/user-guides/server.md

```diff
@@ -487,117 +487,93 @@ r = c.encode(
 
-[//]: # (A minimum YAML file `flow.yml` is as follows:)
+You need a YAML file to config the `clip_server` executor in the Flow in order to deploy. 
+The executors are hosted on [Jina Hub](https://hub.jina.ai) and are automatically sync with `clip_server` Python module. 
+We currently support [PyTorch-backed CLIP](https://hub.jina.ai/executor/gzpbl8jh) and [ONNX-backed CLIP](https://hub.jina.ai/executor/2a7auwg2).
 
-[//]: # ()
-[//]: # (```yaml)
+A minimum YAML file is as follows:
 
-[//]: # (jtype: Flow)
+````{tab} pytorch-flow.yml
 
-[//]: # (executors:)
-
-[//]: # (  - name: CLIPTorchEncoder # The name of the encoder)
-
-[//]: # (    uses: jinahub+docker://CLIPTorchEncoder)
-
-[//]: # (```)
-
-[//]: # ()
-[//]: # (```{warning})
-
-[//]: # (All Executors' `uses` must follow the format `jinahub+docker://MyExecutor` &#40;from [Jina Hub]&#40;https://hub.jina.ai&#41;&#41; to avoid any local file dependencies.)
-
-[//]: # (```)
-
-[//]: # ()
-[//]: # (To deploy,)
-
-[//]: # ()
-[//]: # (```bash)
-
-[//]: # ($ jc deploy flow.yml)
-
-[//]: # (```)
-
-[//]: # ()
-[//]: # (Here `jc deploy` is the command to deploy a Jina project to JCloud.)
-
-[//]: # (Learn more about [JCloud usage]&#40;https://docs.jina.ai/fundamentals/jcloud/&#41;.)
-
-[//]: # ()
-[//]: # ()
-[//]: # (The Flow is successfully deployed when you see:)
-
-[//]: # ()
-[//]: # (```{figure} images/jc-deploy.png)
-
-[//]: # (:width: 60%)
-
-[//]: # ()
-[//]: # (```)
-
-[//]: # ()
-[//]: # (After deploying on jcloud, you can connect to it via client by setting  `grpcs://` as follows:)
-
-[//]: # ()
-[//]: # ()
-[//]: # (```python)
-
-[//]: # (from clip_client import Client)
-
-[//]: # ()
-[//]: # (c = Client&#40;'grpcs://174eb69ba3.wolf.jina.ai'&#41;  # This is the URL you get from previous step)
-
-[//]: # ()
-[//]: # (r = c.encode&#40;)
-
-[//]: # (    [)
-
-[//]: # (        'First do it',)
-
-[//]: # (        'then do it right',)
-
-[//]: # (        'then do it better',)
-
-[//]: # (        'https://picsum.photos/200',)
-
-[//]: # (    ])
-
-[//]: # (&#41;)
+```yaml
+jtype: Flow
+executors:
+  - name: CLIPTorchEncoder
+    uses: jinahub+docker://CLIPTorchEncoder
+```
+````
+````{tab} onnx-flow.yml
 
-[//]: # (print&#40;r&#41;)
+```yaml
+jtype: Flow
+executors:
+  - name: CLIPOnnxEncoder
+    uses: jinahub+docker://CLIPOnnxEncoder
+```
+````
 
-[//]: # (```)
 
-[//]: # ()
-[//]: # (will give you:)
+```{warning}
+All Executors' `uses` must follow the format `jinahub+docker://MyExecutor` (from [Jina Hub](https://hub.jina.ai)) to avoid any local file dependencies.
+```
 
-[//]: # ()
-[//]: # (```text)
+To deploy,
 
-[//]: # ([[ 0.03480401 -0.23519686  0.01041038 ... -0.5229086  -0.10081214)
+````{tab} PyTorch-backed
+```bash
+$ jc deploy pytorch-flow.yml
+```
+````
+````{tab} ONNX-backed
+```bash
+$ jc deploy onnx-flow.yml
+```
+````
```

Comment:
Please also explain the executor share the same parameters as in clip_server, and add the link to `user-guides/server/#clip-model-config`